.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.18
```

Comment:
Sadly, I _think_ this will result in Go 1.18.0, not "the latest 1.18.x".

Reply:
Yeah, you're correct - it only enforces the `major.minor` and doesn't care about the patch. However, this is a good step forward to where we didn't enforce any version.

There's an open PR in asdf to try to tackle it but there's mixed opinion on whether it should be part of the core tool or the plugins.

Are you okay with this to start with, then if the asdf supports 'latest' patches we move to that?